docs/src/rules/no-restricted-globals.md

```diff
@@ -107,6 +109,90 @@ function onClick() {
 
 :::
 
+### globals
+
+An object option whose value is an array containing the names of the globals you want to restrict.
+
+Examples of **incorrect** code for `"event"` and `"fdescribe"` global variable names:
+
+::: incorrect
+
+```js
+/*global event, fdescribe*/
+/*eslint no-restricted-globals: ["error", { globals: ["event", "fdescribe"] }]*/
+
+function onClick() {
+    console.log(event);
+}
+
+fdescribe("foo", function() {
+});
+```
+
+:::
+
+Custom messages for a particular global can also be specified in `globals` array using objects with `name` and `message`:
+
+Examples of **incorrect** code for an `"event"` global variable name, along with a custom error message:
+
+::: incorrect
+
+```js
+/*global event*/
+/* eslint no-restricted-globals: ["error", { globals: [{ name: "event", message: "Use local parameter instead." }] }] */
+
+function onClick() {
+    console.log(event);
+}
+```
+
+:::
+
+### checkGlobalObjectAccess
```

Comment:
Maybe we can shorten this to `checkGlobalObject`? I'm not sure "Access" is adding anything.